github-for-jira.sd.yml

```diff
@@ -630,6 +656,19 @@ environmentOverrides:
             Period: 300
             ComparisonOperator: GreaterThanThreshold
             Statistic: Maximum
+          DLQAlarmOnTooManyMessages:
+            MetricName: ApproximateNumberOfMessagesVisible
+            Namespace: AWS/SQS
+            Description: "Message count for deployment dead letter queue too high!"
+            Threshold: 5000 # number is initially high until we start bringing numbers down
```

Comment:
Then maybe we don't have to add the alarm right now? Or we are supposed to add the alarm and be like super panic when the number did reach 5000?

Reply:
its always got a lot in that queue, sits between 2000-4900 for the last few months, so over 5000 would be a "spike" - but there is a bigger issue here that the queue size is so massive to begin with